charts/sn-console/templates/console/streamnative-console-statefulset.yaml

```diff
@@ -164,6 +164,10 @@ spec:
           - name: BACKEND_DEFAULT_SUPER_USER_ROLE
             value: {{ .Values.configData.BACKEND_DEFAULT_SUPER_USER_ROLE }}
           {{- end }}
+          - name: CONNECTOR_ENABLED
+            value: "{{ .Values.configData.CONNECTOR_ENABLED | default "true" }}"
+          - name: FLINK_SQL_ENABLED
+            value: "{{ .Values.configData.FLINK_SQL_ENABLED | default "false" }}"
```

Comment:
shall we also add FLINK_SQL_ENABLED to value file?

Reply:
Yes, I think it's okay because we want some users to be able to support flink clusters very easily on the console after they've been deployed, but this feature is not directly exposed to users by default

Reply:
Should we avoid this kind of ghost value? Thinking about whether we should have all `.Values` references correspond to the `values.yaml` file one by one?

Reply:
As mentioned above, it is possible to expose the FLINK_SQL_ENABLED option to the user is not suitable, because the sn-platform does not support creating a flink cluster.